index.bs

```diff
@@ -5565,20 +5551,83 @@ WebDriver [=extension command=] is defined as follows:
     </table>
 </figure>
 
+The <dfn>Authenticator Configuration</dfn> is a JSON [=Object=] containing the following parameters:
+
+<figure id="table-addVirtualAuthenticator" class="table">
+    <table class="data">
+        <thead>
+            <tr>
+                <th>Key</th>
+                <th>Value Type</th>
+                <th>Valid Values</th>
+                <th>Default</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>|protocol|</td>
+                <td>string</td>
+                <td>`"u2f"`, `"ctap2"`</td>
+                <td>None</td>
+            </tr>
+            <tr>
+                <td>|transport|</td>
+                <td>string</td>
+                <td>{{AuthenticatorTransport}} values</td>
+                <td>None</td>
+            </tr>
+            <tr>
+                <td>|hasResidentKey|</td>
+                <td>boolean</td>
+                <td>[TRUE], [FALSE]</td>
+                <td>[FALSE]</td>
+            </tr>
+            <tr>
+                <td>|hasUserVerification|</td>
+                <td>boolean</td>
+                <td>[TRUE], [FALSE]</td>
+                <td>[FALSE]</td>
+            </tr>
+            <tr>
+                <td>|isUserPresent|</td>
+                <td>boolean</td>
+                <td>[TRUE], [FALSE]</td>
+                <td>[TRUE]</td>
+            </tr>
+            <tr>
+                <td>|isUserVerified|</td>
+                <td>boolean</td>
+                <td>[TRUE], [FALSE]</td>
+                <td>[TRUE]</td>
```

Comment:
Should the default for |isUserVerified| ought to match the default for |hasUserVerification| ?